shopify_python/__init__.py

```diff
@@ -2,4 +2,12 @@
 # Use of this source code is governed by a MIT-style license that can be found in the LICENSE file.
 from __future__ import unicode_literals
 
+from pylint import lint
+from shopify_python import google_styleguide
+
+
 __version__ = '0.0.0'
```

Comment:
Not really for this PR, but we should have a process for deciding how and when to bump versions.